src/requestor/arborist.py

```diff
@@ -79,6 +81,16 @@ def get_auto_policy_id_for_resource_path(resource_path: str) -> str:
     return policy_id
 
 
+async def user_has_policy(
+    arborist_client: ArboristClient, username: str, policy_id: str
+) -> bool:
+    user = await arborist_client.get_user(username)
+    for policy_data in user["policies"]:
```

Comment:
What are your thoughts on using `any(policy_id==policy_data["policy"] for policy_id in user["policies"])` instead of `return True` and `return False`?

Reply:
that would work too